src/start.ts

```diff
@@ -104,7 +105,8 @@ export async function getAllIssueActivity(issueParams: IssueParams) {
   return mixedEventsAndComments;
 }
 
-export async function getTimelineUsers(issueParams: IssueParams): Promise<GitHubUser[]> {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+async function getTimelineUsers(issueParams: IssueParams): Promise<GitHubUser[]> {
```

Comment:
Any reason we have args we don't use here?

Reply:
@gentlementlegen a good question. I assumed this was added for future use but if it is not needed then it can be removed completely.

Reply:
I mean that the function itself has only dev purposes but the vars inside are used. Does this silence the Knip error?